spec/recipes/omnibus_toolchain_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'spec_helper'
 
 describe 'omnibus::_omnibus_toolchain' do
-  let(:chef_run) { ChefSpec::SoloRunner.converge(described_recipe) }
+  let(:chef_run) {
+    ChefSpec::SoloRunner.new do|node|
+      node.automatic['kernel']['machine'] = 'x86_64'
```

Comment:
To get test coverage I do not need to specifically test the `powerpc` case - just 32 vs 64 bit